runtime/doc/lsp.txt

```diff
@@ -960,12 +960,12 @@ buf_diagnostics_virtual_text({bufnr}, {diagnostics})
                                  *vim.lsp.util.buf_diagnostics_signs()*
 buf_diagnostics_signs({bufnr}, {diagnostics})
                 Place signs for each diagnostic in the sign column.
-                Sign characters can be customized with the following options:
+                Sign characters can be customized with the following commands:
 >
-let g:LspDiagnosticsErrorSign = 'E'
-let g:LspDiagnosticsWarningSign = 'W'
-let g:LspDiagnosticsInformationSign = 'I'
-let g:LspDiagnosticsHintSign = 'H'
+sign define LspDiagnosticsErrorSign text=E
+sign define LspDiagnosticsWarningSign text=W
+sign define LspDiagnosticsInformationSign text=I
+sign define LspDiagnosticsHintSign text=H
```

Comment:
These also need a `texthl` right?

I guess maybe it doesn't matter but it would make it more clear. 

Maybe we could exactly what the command is to generate the defaults here (so it's obvious what they will be)